README.md

```diff
@@ -1 +1,38 @@
-# angular-what-is-an-expression-readme
\ No newline at end of file
+# What is an Expression?
+
+## Overview
+
+Let's take a look at expressions. We've used expressions in our labs without even noticing it!
+
+## Objectives
+
+- Describe expressions
+- Using an expression
+- Conditionally assign data using the ternary operator
+- Create an expression to display length of data
+
+## Express yourself
+
+Anything in between the `{{}}` braces we've been using in our templates is an "expression". We've only been doing *simple* expressions up to this point, just referencing a variable.
+
+However, expressions can be a lot more powerful. We can add, look up indexes/properties in arrays/objects or even use ternary operators.
```

Comment:
in theory, could we use expressions to output any javascript evaluation? would {{ 1 + 1 }} output 2?